docs/porting/target/cmake.md

```diff
@@ -0,0 +1,76 @@
+# CMake target porting
+
+## Add CMakeLists.txt
+
+Add `CMakeLists.txt` to folders where target's files are located. Please follow logical directory structure (do not create more `CMakeLists.txt` than are required). If there is 3rd party driver, it should have own `CMakeLists.txt`.
+
+## Add your sources and includes
+
+New directory is added using  `add_subdirectory`. You can use conditional inclusion based on
+`MBED_TARGET_LABELS` or `MBED_TOOLCHAIN` or similar available CMake variables.
+
+```
+# add sdk_driver only for specific target
+if("NEW_TARGET" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(sdk_driver)
+endif()
+
+add_subdirectory(driver_always_added)
+```
+
+Sources are added using `target_sources` where all code files must be added for `mbed-core` library.
+
+```
+target_sources(mbed-core
```

Comment:
Is this an example? if so, indicate so

Reply:
I don't follow, indicate? All code snippets in this porting guide are for illustration, snippets taken from real cmake files but edited to make a point.

Reply:
Ok. It's probably cosmetic point, but consider adding `For example: ...` before the code.

Reply:
We do not do that in our docs, I've checked porting guide: https://os.mbed.com/docs/mbed-os/v6.4/porting/porting-the-hal-apis.html

It's expected the code snippets are not real code and must be adjusted by a user who is porting.